JSON/requirements-sample-fio.json

```diff
@@ -14,12 +14,15 @@
         ]
     },
     "validations": {
-        "size_KMG": {
-            "description": "bytes in k/K (1024), m/M (1024^2) or g/G (1024^3): 4k 16M 1g",
+        "size_BKMG": {
+            "description": "bytes in B/K/M/G, converted to K, shown in KB",
             "args": [ "bs" ],
-            "vals": "[0-9]+[kbmgKBMG]",
-            "transform": [ "s/([0-9]+)[gG]/($1*1024).\"M\"/e",
-                           "s/([0-9]+)[mM]/($1*1024).\"K\"/e" ]
+            "vals": "[0-9]+[BKMG]?",
+            "convert": "",
```

Comment:
I thought this would have to be `"convert": "KB"`?

Reply:
yes, it must be "K", sorry.